tests/isoObject.cpp

```diff
@@ -0,0 +1,371 @@
+#include <gtest/gtest.h>
+#include <thread>
+#include <boost/asio.hpp>
+#include <boost/system/system_error.hpp>
+#include "iso22133object.hpp"
+extern "C" {
+	#include "iso22133.h"
+}
+
+
+using namespace ISO22133;
+using namespace boost::asio;
+using namespace std::chrono_literals;
+
+struct timeval * TimeSetToCurrentSystemTime(struct timeval *time)
+{
+	// Get system time
+	struct timespec currentTime;
+	clock_gettime(CLOCK_REALTIME, &currentTime);
+	time->tv_sec = currentTime.tv_sec;
+	time->tv_usec = currentTime.tv_nsec / 1000;
+	return time;
+}
+
+class TCPConnection
```

Comment:
As I understand this is more of a ATOS emulator? I find it a bit confusing to have a UDP socket inside a class called TCPconnection

Reply:
^ I second this.

Reply:
renamed